src/main/java/lv/ctco/javaschool/goal/boundary/GoalApi.java

```diff
@@ -91,8 +85,9 @@ public void createNewGoal(GoalFormDto goalDto) {
         }
     }
 
+
     Set<Tag> parseStringToTags(String value) {
```

Comment:
parse function shouldn't be in API class.
change function name(doesn't do what meant)